features/step_definitions/nav_steps.rb

```diff
@@ -6,6 +6,10 @@
   expect(page.current_path).to eq path(page_name)
 end
 
+Then(/^I should be redirected to the "([^"]*)" page$/) do |page_name|
+  expect(page.current_path).to eq path(page_name)
```

Comment:
@MikaelFeher I thought you could do `expect(response).to redirect_to root_path` but that doesn't seem to work. We can leave this out for now and merge this. Any objections to this PR @tochman?